cpp/ps0001/ps0001_sol.cpp

```diff
@@ -4,15 +4,19 @@ using namespace std;
 
 int main()
 {
-    int a ,b ,c ;
-    int s ;
-    cout<< "Enter first number \n" ;
-    cin>> a;
-    cout<< "Enter second number \n" ;
-    cin>> b ;
-     cout<< "Enter third number \n" ;
-     cin>> c ;
-     s= a + b + c ;
-     cout<< " The sum is "<< s << endl ;
+    // prompting users for 3 numbers to add 1/2
+    unsigned short a, b, c;
+    unsigned short s;
+    cout << "Enter first number \n";
+    cin >> a;
+    cout << "Enter second number \n";
+    cin >> b;
+    cout << "Enter third number \n";
+    cin >> c;
+    

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add